src/new/templates.js

```diff
@@ -9,53 +9,56 @@ export function api(apiName)  {
 }`
 }
 
-export function env(env)  {
+export function env (env) {
   return `module.exports = {
   env: "${env}",
   secretkey: "${env}-secret-key"
 }`
 }
 
-export function gitignore()  {
+export function gitignore () {
   return `dist/*
 node_modules/*
-functions/*/node_modules/*
 config/*`
 }
 
-export function lambda(){
+export function lambda () {
   let obj = {
-    Handler: "index.handler",
+    Handler: 'index.handler',
     MemorySize: 128,
-    Role: "",
+    Role: '',
     Timeout: 10,
-    Runtime: "nodejs4.3"
+    Runtime: 'nodejs4.3'
   }
 
   return JSON.stringify(obj, null, 2)
 }
 
-export function pkg(apiName, api)  {
+export function pkg (apiName) {
   let obj = {
     name: apiName,
-    version: "1.0.0",
+    version: '1.0.0',
     private: true,
-    description: "",
-    license: "",
-    dependencies: {},
-    shep: {}
+    description: '',
+    license: '',
+    devDependencies: {
+      webpack: '2.1.0-beta.25'
+    },
+    shep: {
+      region: '',
+      accountId: '',
+      apiId: ''
+    }
   }
 
-  if (api === false) { obj.shep.api = false }
-
   return JSON.stringify(obj, null, 2)
 }
 
-export function readme(apiName){
+export function readme (apiName) {
   return `# ${apiName}`
 }
 
-export function webpack(){
+export function webpack () {
```

Comment:
The reason I went the route above is because its nice to have a real clean break between shep and webpack. If your stuff isn't working or building you can always just run `webpack` yourself and know your getting the exact result same result as shep.

If we go the route you suggest will is still be easy to run `webpack <args>`? I'm def open too it as long as `<args>` doesn't get too complex and we can document how it all works for end users.

cc @jqyu who has also done a ton of webpack stuff and might have an opinion.